lua/wikis/fighters/Infobox/League/Custom.lua

```diff
@@ -46,11 +45,14 @@ function CustomLeague.run(frame)
 	args.circuitabbr = args.circuitabbr or CustomLeague.getAbbrFromSeries(args.circuit)
 
 	-- Auto Icon
-	local seriesIconLight, seriesIconDark = CustomLeague.getIconFromSeries(args.series)
-	args.circuitIconLight, args.circuitIconDark = CustomLeague.getIconFromSeries(args.circuit)
-	args.icon = args.icon or seriesIconLight or args.circuitIconLight
-	args.icondark = args.icondark or seriesIconDark or args.circuitIconDark
-	args.display_series_icon_from_manual_input = MANUAL_SERIES_ICON
```

Comment:
why remove this line?

Reply:
re-added, I may have overdeleted it (the manual series line)